netaddr.go

```diff
@@ -323,3 +325,94 @@ func (p IPPort) TCPAddr() *net.TCPAddr {
 		Zone: zone,
 	}
 }
+
+// IPPrefix is an IP address prefix representing an IP network.
+//
+// The first Bits of IP are specified, the remaining bits match any address.
+// The range of Bits is [0,32] for IPv4 or [0,128] for IPv6.
```

Comment:
What's the meaning of 0 for bits? Move the Bits docs to the field below?

Reply:
/0 can be used to indicate the entire address space, e.g. `0.0.0.0/0` and `::/0`. It's unusual but valid.

Reply:
Test for that then.